src/components/InputForm.js

```diff
@@ -54,10 +57,10 @@ const InputBooks = () => {
           <option>Thriller</option>
           <option>Drama</option>
         </select>
-        <button type="submit">Add Book</button>
+        <button type="submit" className="addBook">Add Book</button>
```

Comment:
- [ ] Kindly use the correct button text color so that it matches the Zeplin design.

**This is how your button looks like:**

![image](https://user-images.githubusercontent.com/86318284/202713793-731403f8-138f-41a4-a247-8a0b5ddefb3c.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/202713851-16f72004-c1c6-4312-b43e-4f885630c5fc.png)